### ISSUE TITLE
Test failure Magick++/tests/tests.tap

### ISSUE THREAD

**Issue by remicollet** [ImageMagick/ImageMagick6#349]
### ImageMagick version

6.9.13-18

### Operating system

Linux

### Operating system, version and so on

RHEL 8.10

### Description

On RHEL 8.10

```
============================================================================
Testsuite summary for ImageMagick 6.9.13-18
============================================================================
# TOTAL: 76
# PASS:  75
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0
============================================================================
See ./test-suite.log
Please report to https://github.com/ImageMagick/ImageMagick6/issues
============================================================================
```

From test-suite.log 
```
Caught exception: lt-coderInfo: unable to load module `/builddir/build/BUILD/ImageMagick-6.9.13-18/coders/tiff.la': file not found @ error/module.c/OpenModule/1319
not ok
FAIL: Magick++/tests/tests.tap 5
```

Fill test-suite.log attached
[test-suite.txt](https://github.com/user-attachments/files/17713168/test-suite.txt)

### Steps to Reproduce

Run test suite on RHEL-8

### Images

N/A

**Comment by remicollet** [comment]
Notice: 
- everything OK on RHEL 9.5 and Fedora >= 39
- everything OK with ImageMagic **7.1.1-40**

**Comment by remicollet** [comment]
Seems fixed in 6.9.13-41
